openbb_platform/providers/nasdaq/tests/test_nasdaq_app.py

```diff
@@ -0,0 +1,784 @@
+                                            "ex_dividend_date",
+                                            "symbol",
+                                            "amount",
+                                            "name",
+                                            "record_date",
+                                            "payment_date",
+                                            "declaration_date",
+                                            "annualized_amount",
+                                        ]
+                                    },
+                                }
+                            },
+                        },
+                    }
+                ],
+            },
+            "ipo-calendar": {
+                "id": "ipo-calendar",
+                "name": "IPO Calendar",
+                "layout": [
+                    {
+                        "i": "nasdaq_ipo_calendar",
+                        "x": 0,
+                        "y": 2,
+                        "w": 40,
+                        "h": 21,
+                        "state": {
+                            "chartView": {"enabled": False, "chartType": "line"},
+                            "columnState": {
+                                "default": {
+                                    "columnPinning": {
+                                        "leftColIds": ["ipo_date", "symbol", "name"],
+                                        "rightColIds": [],
+                                    },
+                                    "columnOrder": {
+                                        "orderedColIds": [
+                                            "ipo_date",
+                                            "symbol",
+                                            "name",
+                                            "offer_amount",
+                                            "share_count",
+                                            "share_price",
+                                            "deal_status",
+                                            "exchange",
+                                            "id",
+                                        ]
+                                    },
+                                }
+                            },
+                        },
+                    }
+                ],
+            },
+            "economic-calendar": {
+                "id": "economic-calendar",
+                "name": "Economic Calendar",
+                "layout": [
+                    {"i": "nasdaq_economic_calendar", "x": 0, "y": 2, "w": 40, "h": 20}
+                ],
+            },
+        },
+        "groups": [],
+    }
+    route = [d for d in nasdaq_app.routes if d.path == "/apps.json"][0]
+    response = await route.endpoint()
+    assert isinstance(response, dict)
+    assert response == expected
+
+
+def test_widgets_json(nasdaq_app):
+    expected = {
```

Comment:
These expected values make up a lot of lines in the actual test file. It makes it a bit harder to follow "what is actually being tested".

Please consider moving them to a distinct test data files/collection (similar to how we record cassettes with vcrpy)